ui/packages/components/src/CodeBlock/CodeBlock.tsx

```diff
@@ -133,15 +196,6 @@ export function CodeBlock({ header, tabs, actions = [] }: CodeBlockProps) {
     setActiveTab(index);
   };
 
-  function handleEditorDidMount(editor: MonacoEditorType) {
```

Comment:
This was the cause of the broken borders as it was not adding this class when there were more than one editor on the page. So instead I just removed it and applied this globally via css, see above.